bin/api.wsgi

```diff
@@ -21,6 +22,8 @@ ap.add_argument('--central_uri', help='scitran central api', default='https://sd
 ap.add_argument('--log_level', help='log level [info]', default='info')
 ap.add_argument('--drone_secret', help='shared drone secret')
 
+# REVIEW: remove most above flags?
+ap.add_argument('--config', help='path to config file')
```

Comment:
Based on our discussion I think we're leaving all / most current flags.
If that's the case I'll remove this comment.